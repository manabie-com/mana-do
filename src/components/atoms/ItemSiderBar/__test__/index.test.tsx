import React from 'react';
import {render} from '@testing-library/react';
import {ItemSiderBar} from "../index";


describe('<ItemSiderBar /> render', () => {
    it('should match snapshot', function () {
        const props = {
            iconName: 'star',
            label:'Important',
            isMenuAcive: true
        }
        const Element = render(<ItemSiderBar {...props}/>);
        expect(Element.container.firstChild).toMatchSnapshot();
    });
})