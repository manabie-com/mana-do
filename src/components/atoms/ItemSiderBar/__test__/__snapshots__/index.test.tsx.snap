// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ItemSiderBar /> render should match snapshot 1`] = `
<div
  class="WrapperItemSiderBar"
>
  <div
    class="ItemSiderBar--acitve ItemSiderBar"
  >
    <i
      class="manabie-star"
    />
    <span>
      Important
    </span>
  </div>
</div>
`;
